content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -9,135 +9,20 @@ disable_sidebar: true
 <div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
 {{< /site-region >}}
 
-<div class="alert alert-info">
-The features described on this page are in beta. Contact your Customer Success Manager to provide feedback or ask for help.
-</div>
-
-## Supported Oracle versions, features, and architectures
-
-- **Versions**: 11.2, 12c, 18c, 19c, and 21c
-- **Deployment configurations**: Self-managed, RDS, RAC, Exadata, Autonomous Database
-- **Architecture**: Multi-tenant, non-CDB, RDS single-tenant
```

Comment:
Some slight adjustments for consistency with internal feature announcement:

- **Versions**: 11g, 12c, 18c, 19c, 21c
- **Deployment configurations**: Self-managed, RDS, RAC, Exadata, Autonomous Database, Automatic Storage Management
- **Architectures**: Multi-tenant, non-CDB

Reply:
@caseycull The above content was removed in favor of the table below. I can add the deployment configurations and architectures back in below the table, does that work?